src/backend/libpq/auth.c

```diff
@@ -2554,6 +2554,7 @@ static int
 InitializeLDAPConnection(Port *port, LDAP **ldap)
 {
 	int			ldapversion = LDAP_VERSION3;
+	int			ldaprestart = 1;
```

Comment:
According to https://linux.die.net/man/3/ldap_set_option for LDAP_OPT_RESTART "value should either be LDAP_OPT_OFF or LDAP_OPT_ON". I suggest to use LDAP_OPT_ON instead of 1

Reply:
applied